File: LeetCode Solutions/Problems/1550. Three Consecutive Odds.java
class Solution {
    public boolean threeConsecutiveOdds(int[] arr) {
        if(arr.length < 3){
            return false;
        }
        for(int i=0; i<arr.length-2; i++){
            int odd1 = arr[i], odd2 = arr[i+1], odd3 = arr[i+2];
            if((odd1%2 !=0) && (odd2%2 !=0) && (odd3%2 !=0)){
                return true;
            }
        }
        return false;
    }
}

File: LeetCode Solutions/Problems/1550. Three Consecutive Odds.py
class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:
        if(len(arr) < 3):
            return False
        for i in range(len(arr)-2):
            odd1, odd2, odd3 = arr[i], arr[i+1], arr[i+2]
            if((odd1%2 !=0) and (odd2%2 !=0) and (odd3%2 !=0)):
                return True
        return False